SlackAPI/SlackClient.cs

```diff
@@ -721,75 +637,13 @@ public void UploadFile(Action<FileUploadResponse> callback, byte[] fileData, str
 
             parameters.Add(string.Format("{0}={1}", "channels", string.Join(",", channelIds)));
 
-            using(HttpClient client = new HttpClient())
```

Comment:
There is lot of discussion of Internet about HttpClient but it seems it's better to use a single HttpClient instance